src/commands/template

```diff
@@ -30,7 +30,10 @@ extra_params=''
 # Kubectl can take some time before stdin is available for reading
 # thats why we check if just before it's required
 if read -t "${STDIN_WAIT_S}"; then
-  cat <&0 > "${tmpfile}"
+  # Save first line read from stdin
+  echo "${REPLY}" > "${tmpfile}"
```

Comment:
Maybe comment this so that it's clear to the reader that `${REPLY}` comes from `read` call

ref: https://linuxcommand.org/lc3_man_pages/readh.html
>     If no NAMEs are supplied, the line read is stored in the REPLY variable.